examples/auth/index.js

```diff
@@ -63,13 +62,22 @@ function authenticate(name, pass, fn) {
   // apply the same algorithm to the POSTed password, applying
   // the hash against the pass / salt, if there is a match we
   // found the user
-  hash({ password: pass, salt: user.salt }, function (err, pass, salt, hash) {
+
+
+//  ------------------ A better Way to Hash Password without passing any salt using Bcrypt.Js----------------- 
+  if(pass){
+  if (bcrypt.compareSync(pass, encryPassword)) {
```

Comment:
You probably shouldn't be using sync calls in a web server, especially when there is an async alternative provided in the API.